effekt/shared/src/main/scala/effekt/lifted/Tree.scala

```diff
@@ -53,7 +53,7 @@ enum Extern {
   // WARNING: builtins do not take evidence. If they are passed as function argument, they need to be eta-expanded.
   //   (however, if they _would_ take evidence, we could model mutable state with this)
   // TODO revisit
-  case Def(id: Id, tparams: List[Id], params: List[Param], ret: ValueType, bodies: ExternBody)
+  case Def(id: Id, tparams: List[Id], params: List[Param], ret: ValueType, control: Boolean, body: ExternBody)
```

Comment:
This will not be necessary when rebasing, since core has annotated captures already.